crates/client/src/config.rs

```diff
@@ -68,14 +77,17 @@ pub struct Config {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub default_url: Option<String>,
 
+    /// List of registries
+    pub registries: Vec<String>,
```

Comment:
I can't seem to determine what this is used for. Could you describe the intention behind this field?

Reply:
Bump @macovedj

Reply:
Peter's comment helped me realize it wasn't necessary.  Went ahead and deleted this field.